### ISSUE TITLE
Show next feed after all items in current feed are marked as read

### ISSUE THREAD

**Issue by stephenlhorton** [Ashinch/ReadYou#738]
**1. Environment**

-   Device: Pixel 6 Pro
-   OS: Android 15 Beta 2.2
-   APP: 0.10.1

**2. Describe the feature request**

After all items in a feed are marked as read, automatically proceed to the next feed. Currently, you have to back out of the feed before preceding to the next one.

**Comment by bzeiss** [comment]
I was about to ask for a similar enhancement, but wanted to suggest to directly return to the feeds list instead of stays on an empty feed. Looking at an empty feed is pretty pointless and just one unecessary tap to the back button. It disrupts the reading flow. However, I wouldn't want to jump directly to the next feed as I don't read in order of the feeds, but by interest. Thus, I'd want to get back to the feed list and choose what to read next.

As there seem to be different preferences with respect to this behavior, a configuration switch what to do next is probably the best choice.

**Comment by stephenlhorton** [comment]
I agree. A setting that gives the option of what to do after all items in a feed are marked read is probably the best approach.
